src/delphi/vcl.castlecontrol.reg.pas

```diff
@@ -0,0 +1,45 @@
+{
+  Copyright 2022-2023 Michalis Kamburelis.
+
+  This file is part of "Castle Game Engine".
+
+  "Castle Game Engine" is free software; see the file COPYING.txt,
+  included in this distribution, for details about the copyright.
+
+  "Castle Game Engine" is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+  ----------------------------------------------------------------------------
+}
+
+{ Control with OpenGL context on a Delphi VCL form. }
+unit vcl.castlecontrol.reg;
```

Comment:
Make unit names use `PascalCase` with dots, so `Vcl.CastleControl.Reg`.

Moreover, to make the unit suitable for possible other design-time stuff, please call it just `Vcl.CastleControl.Design`.